### PR TITLE
feat: remove published state

### PR DESCRIPTION
**Pull request by jianwei1**
# Description

### <samp>🤖 Generated by Copilot at b12b08c</samp>

This pull request removes some unnecessary elements from the journey view components, such as the publish menu item and the status icon. These changes are part of a redesign of the journey list view, which now provides a clearer and simpler way to publish and manage journeys.

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/33281894/todos/6405610925)

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] it should not show publish button in journey details page menu

# Walkthrough

### <samp>🤖 Generated by Copilot at b12b08c</samp>

* Remove publishing menu item from journey view ([link](https://github.com/JesusFilm/core/pull/1820/files?diff=unified&w=0#diff-2c9f44f9db6ec77ee7fdf06b593abfe7aab0fc14735b41b66094629ff25bc082L208-L218))
* Remove status icon and text from journey details ([link](https://github.com/JesusFilm/core/pull/1820/files?diff=unified&w=0#diff-098446e49a8e222b78a77bdd30a6a8b3816b3f9b4eb450523510b5647fdf0dc3L64-L78))

### CODE REVIEW

**Review by csiyang — CHANGES_REQUESTED** [review]

File: apps/journeys-admin/src/components/JourneyView/Properties/JourneyDetails/JourneyDetails.tsx [hunk 1]
```diff
@@ -28,54 +23,12 @@ export function JourneyDetails({
...
-
   return (
     <>
       {journeyType === 'Template' ? (
         <Language isPublisher={isPublisher} />
       ) : (
         <>
-          <Box sx={{ display: 'flex', flexDirection: 'row' }}>
-            {journey != null && currentStatus != null ? (
-              <>{currentStatus.icon}</>
```
[the remark this answers is not in the document]
Comment (Kneesal) [reply apps/journeys-admin/src/components/JourneyView/Properties/JourneyDetails/JourneyDetails.tsx:66]: Wasn't in the scope of the ticket but I can clarify with Vlad

File: apps/journeys-admin/src/components/JourneyView/Menu/Menu.tsx [hunk 2]
```diff
@@ -97,30 +78,7 @@ export function Menu(): ReactElement {
...
-          variant: 'success',
-          preventDuplicate: true
-        })
-      : enqueueSnackbar('Journey Published', {
-          variant: 'success',
-          preventDuplicate: true
-        })
-  }
   const handleTemplate = async (teamId: string | undefined): Promise<void> => {
     if (journey == null || teamId == null) return
```
[the remark this answers is not in the document]
Comment (Kneesal) [reply apps/journeys-admin/src/components/JourneyView/Menu/Menu.tsx:84]: Wasn't in the scope of the ticket but I can clarify with Vlad

[the remark this answers is not in the document]
Comment (Kneesal) [reply apps/journeys-admin/src/components/JourneyView/Menu/Menu.tsx:84]: as per vlad's comment I will leave this untouched

File: apps/journeys-admin/src/components/JourneyView/Properties/JourneyDetails/JourneyDetails.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Kneesal) [reply apps/journeys-admin/src/components/JourneyView/Properties/JourneyDetails/JourneyDetails.tsx:66]: Will remove this

File: apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardInfo/JourneyCardInfo.tsx [hunk 3]
```diff
@@ -76,7 +75,7 @@ export function JourneyCardInfo({ journey, variant }: Props): ReactElement {
         <>
           {journey != null ? (
             <>
-              <StatusChip status={journey.status} />
+              {/* <StatusChip status={journey.status} /> */}
```
Comment (csiyang) [inline apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardInfo/JourneyCardInfo.tsx:78]: remove
```suggestion
```

File: apps/journeys-admin/src/components/JourneyView/Menu/Menu.spec.tsx [hunk 4]
```diff
@@ -56,14 +56,14 @@ describe('JourneyView/Menu', () => {
     expect(menu).toHaveAttribute('aria-expanded', 'true')
   })
 
-  it('should preview if journey is published', () => {
+  it('should preview if journey', () => {
```
Comment (csiyang) [inline apps/journeys-admin/src/components/JourneyView/Menu/Menu.spec.tsx:59]: fix the title

File: apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardInfo/JourneyCardInfo.tsx [hunk 5]
```diff
@@ -76,7 +75,7 @@ export function JourneyCardInfo({ journey, variant }: Props): ReactElement {
         <>
           {journey != null ? (
             <>
```
Comment (csiyang) [inline apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardInfo/JourneyCardInfo.tsx:77]: remove redundant fragment

**Review by csiyang — CHANGES_REQUESTED** [review]

File: apps/journeys-admin/src/components/JourneyView/Menu/Menu.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (csiyang) [reply apps/journeys-admin/src/components/JourneyView/Menu/Menu.tsx:84]: @Kneesal Vlad's comment only covered templates, this is in journeys. This comment is not yet resolved

**Comment by csiyang** [comment]
Also why is this being merged into another branch and not main?

**Comment by Kneesal** [comment]
> Also why is this being merged into another branch and not main?

so I can test this branch works with it's corresponding backend branch. Once backend branch gets merged first, it will reset to main by default

File: apps/journeys-admin/src/components/JourneyView/Menu/Menu.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Kneesal) [reply apps/journeys-admin/src/components/JourneyView/Menu/Menu.tsx:84]: Vlad has come back and specified he wants the menu button to stay in place as is

**Review by csiyang — DISMISSED** [review]

**Review by csiyang — APPROVED** [review]

**Pull request merged**